src/flask_app.py

```diff
@@ -53,4 +53,4 @@ def server_error(exception):
 	handler.setFormatter(formatter)
 	app.logger.info('data sorting services is running')
 	app.logger.addHandler(handler)
-	app.run('0.0.0.0', 5000)
+	app.run()
```

Comment:
sorry, I forgot to add `'0.0.0.0',5000` back, I have to remove it to be able to test locally.